### PR TITLE
Fix tests to run without tweaking

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -64,10 +64,10 @@
 toxenv = os.environ.get('TOXENV', '')
 
 # Check if PostgreSQL settings are set via environment variables
-pgname = os.environ.get('RESOLWE_POSTGRESQL_NAME', 'resolwe-bio')
-pguser = os.environ.get('RESOLWE_POSTGRESQL_USER', 'postgres')
+pgname = os.environ.get('RESOLWE_POSTGRESQL_NAME', 'resolwe')
```
Comment (dblenkus) [inline tests/settings.py]: Name has to be the same as `POSTGRES_DB` in `docker-compose.yml`.

**Comment by dblenkus** [comment]
Rebase to `master`.

File: docs/CHANGELOG.rst [hunk 2]
```diff
@@ -13,6 +13,7 @@ Changed
 -------
 * Decorate all tests that currently fail on Docker with ``skipDockerFailure``
 * Require Resolwe 1.2.0
+* Renamed DB user in test project
```
Comment (tjanez) [inline docs/CHANGELOG.rst]: Also mention that you changed PostgreSQL port in the test project

Comment (tjanez) [inline docs/CHANGELOG.rst]: Also, please write CHANGELOG entries in the present tense.

File: docs/CHANGELOG.rst [hunk 3]
```diff
@@ -23,6 +24,7 @@ Added
 * ``skipDockerFailure`` test decorator
 * Expand documentation on running tests
 * Use Travis CI to run the tests
+* Added docker-compose command for PostgreSQL
```
Comment (tjanez) [inline docs/CHANGELOG.rst]: Change to `Add docker-compose configuration for PostgreSQL`

**Comment by tjanez** [comment]
@mstajdohar, to fix the tests, you'll need to update `.travis.yml`.

Define [global variable](https://docs.travis-ci.com/user/environment-variables/#Global-Variables) `RESOLWE_POSTGRESQL_PORT` (and `RESOLWE_POSTGRESQL_USER` if things won't work with the new `resolwe` user). Move existing environment variables under `matrix`:
```yaml
env:
  global:
    - RESOLWE_POSTGRESQL_PORT=5432
  matrix:
    - TOX_ENV=py27
    - TOX_ENV=py34
    - TOX_ENV=py35
```

Please, also update the documentation and mention the option of installing Docker and using docker-compose instead of installing PostgreSQL directly on the system.

**Comment by mstajdohar** [comment]
I have addressed all issues except docs. I have to rethink how to structure docs and I plan to do it in a separate PR.

File: docs/CHANGELOG.rst [hunk 4]
```diff
@@ -14,6 +14,9 @@ Changed
 * Decorate all tests that currently fail on Docker with ``skipDockerFailure``
 * Require Resolwe's ``master`` git branch
 * Put packaging tests in a separate Tox testing environment
+* Require Resolwe 1.2.0
```
Comment (tjanez) [inline docs/CHANGELOG.rst]: Please, remove this line. This is a mistake of the otherwise very useful git's 'union' merge driver.

**Comment by tjanez** [comment]
Ok, we can document Docker compose separately, we just need to remember it.

File: docs/CHANGELOG.rst (hunk 4 above)
Comment (mstajdohar) [inline docs/CHANGELOG.rst]: Removed.

**Pull request merged**